### PR TITLE
Change the image tag for openshift-migration-legacy operator

### PR DESCRIPTION
**Pull request by RichardHoch**
MTC 1.8.x; OCP 4.13+ 

Resolves https://issues.redhat.com/browse/MIG-1564 by changing the command in steps 2 and 7 of https://docs.redhat.com/en/documentation/openshift_container_platform/4.13/html/migrating_from_version_3_to_4/upgrading-3-4#migration-upgrading-mtc-with-legacy-operator_upgrading-3-4  .  After consulting with Dylan, it was decided to introduce a new attribute for the MTC legacy image, {mtc-legacy-image} -- whose value is currently 1.7, although the MTC version is 1.8. 

Previews:

a.  https://81481--ocpdocs-pr.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/upgrading-3-4.html#migration-upgrading-mtc-with-legacy-operator_upgrading-3-4 [procedure steps 2 and 7]
b.  https://81481--ocpdocs-pr.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/upgrading-mtc#migration-upgrading-mtc-with-legacy-operator_upgrading-mtc [procedure steps 2 and 7]

@dymurray Is the section in preview b, "Upgrading the Migration Toolkit for Containers on OpenShift Container Platform versions 4.2 to 4.5," which is in the regular MTC user guide,  still relevant?

### CODE REVIEW

**Comment by RichardHoch** [comment]
@dymurray Please review this PR.

**Review by dymurray — APPROVED** [review]

File: _attributes/common-attributes.adoc [hunk 1]
```diff
@@ -60,6 +60,7 @@ endif::[]
 :mtc-full: Migration Toolkit for Containers
 :mtc-version: 1.8
 :mtc-version-z: 1.8.4
+:mtc-legacy-image: 1.7
```
Comment (stillalearner) [inline _attributes/common-attributes.adoc:63]: it should be v1.7

Comment (stillalearner) [reply _attributes/common-attributes.adoc:63]: $ podman pull registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:1.7
Trying to pull registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:1.7...
Error: initializing source docker://registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:1.7: reading manifest 1.7 in registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator: manifest unknown

Comment (stillalearner) [reply _attributes/common-attributes.adoc:63]: $ podman pull registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v1.7
Trying to pull registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v1.7...
Getting image source signatures
Checking if image destination supports signatures
Copying blob 97da74cc6d8f done   | 
Copying blob a904d427d387 done   | 
Copying blob 872602c382da done   | 
Copying blob fdaf21948019 done   | 
Copying blob d8190195889e done   | 
Copying blob a41c226c3b58 done   | 
Copying config 36bdb2f742 done   | 
Writing manifest to image destination
Storing signatures
36bdb2f74202afa5c35253df24211a1c4f9d15c1665cf34b7376ea1c652fdf76

File: modules/migration-upgrading-mtc-with-legacy-operator.adoc [hunk 2]
```diff
@@ -35,8 +35,7 @@ $ podman login registry.redhat.io
 +
 [source,terminal,subs="attributes+"]
 ----
-$ podman cp $(podman create \
-  registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v{mtc-version}):/operator.yml ./
+$ podman cp $(podman create registry.redhat.io/rhmtc/openshift-migration-legacy-rhel8-operator:v{mtc-legacy-image}:/operator.yml ./
 ----
```
Comment (RichardHoch) [inline modules/migration-upgrading-mtc-with-legacy-operator.adoc:39]: @stillalearner the "v" is back.

**Comment by RichardHoch** [comment]
@stillalearner Thanks for catching that! Fixed (see comments).

**Review by stillalearner — APPROVED** [review]
lgtm

**Review by skopacz1 — APPROVED** [review]
peer review lgtm

**Comment by RichardHoch** [comment]
Please do not merge yet, just check -- the release should be GA soon but, but has not happened yet.

**Comment by ShaunaDiaz** [comment]
@RichardHoch Please squash your commits down to 1.

**Comment by ShaunaDiaz** [comment]
> @ShaunaDiaz Commits squashed. Sorry. Is it good for merging? GA is still delayed, but want to know if I need to make any changes.. Thanks.

No, it looks good to me. Placed an ok-to-merge label on for when the time comes.

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.18` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.17. And any PR going into `main` must also target the latest version branch (enterprise-4.18).

If the update in your PR does NOT apply to version 4.18 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Comment by RichardHoch** [comment]
This PR was cleared for merging, but we needed to wait for GA. The version was released, so please merge and publish. Thanks!

**Pull request merged**